apps-devstg/us-east-1/databases-mysql --/config.tf

```diff
@@ -25,11 +26,11 @@ provider "vault" {
 # Backend Config (partial)    #
 #=============================#
 terraform {
-  required_version = ">= 0.14.11"
+  required_version = "~> 1.1.3"
 
   required_providers {
     aws   = "~> 4.0"
-    vault = ">= 2.21.0"
+    vault = "~> 3.6.0"
```

Comment:
@rodriguez-matias Just curious if everything worked as expected with this new provider version.

Reply:
Yes! it works fine, I'll paste the output of `tf apply` for` mysql-databases` back here also.